vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java

```diff
@@ -39,6 +40,7 @@
     private IWaitActions waitActions;
     @Inject private IExpectedConditions<By> expectedConditions;
     @Inject private ElementActions elementActions;
+    @Inject private LocatorUtil locatorUtil;
```

Comment:
please use constructor based injection

Reply:
It's would be good to discuss the strategy of using constructor based DI for classes where the number of params may exceed 5 (or whatever the best practices tell us)

Reply:
If the class has more than five deps possibly it's just a "god-class" and logic should be revised? :)